### PR TITLE
UserMessage component basics

### PR DESCRIPTION
**Pull request by PopDaph**
Very basic first iteration, just so that it can be reused for the AgentMessage if needed. 
Next steps: 
- Handle propertly the udpate on the layout so that we can have the grey background on the full width. EDIT: Actually done on this PR
- Handle markdown. 
- Handle mentions (pending some API work first)

Capture d’écran 2023-09-12 à 18 26 37

### CODE REVIEW

File: front/components/assistant/conversation/Conversation.tsx [hunk 1]
```diff
@@ -1,6 +1,25 @@
+import { Avatar } from "@dust-tt/sparkle";
+
 import { useConversation } from "@app/lib/swr";
+import { UserMessageType } from "@app/types/assistant/conversation";
 import { WorkspaceType } from "@app/types/user";
 
+function UserMessage({ message }: { message: UserMessageType }) {
```
Comment (spolu) [inline front/components/assistant/conversation/Conversation.tsx:7]: Let’s make it a separate component right away?

**Review by philipperolet — CHANGES_REQUESTED** [review]
+1 on making a separate component right away, otherwise lgtm :relaxed: 

Also, question on a container component Message (in which UserMessages and AgentMessages would be) since there is common stuff to be done exactly the same way (avatars, top button positions etc) and preventing divergence would help
But maybe too soon?

**Review by spolu — CHANGES_REQUESTED** [review]

**Comment by spolu** [comment]
Looks great! Left a small change request on who manages the width, I think it should be the Conversation component not the message one.

File: front/components/assistant/conversation/UserMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Avatar } from "@dust-tt/sparkle";
+
+import { UserMessageType } from "@app/types/assistant/conversation";
+
+export function UserMessage({ message }: { message: UserMessageType }) {
+  return (
+    <div className="bg-structure-50 py-6">
+      <div className="mx-auto flex max-w-4xl gap-4 px-6">
```
[the remark this answers is not in the document]
Comment (PopDaph) [reply front/components/assistant/conversation/UserMessage.tsx]: agreed, I added a commit: https://github.com/dust-tt/dust/pull/1418/commits/31bb8354dfda81295090bd130849369a8d7891fd
Good for you?

**Review by spolu — APPROVED** [review]

**Pull request merged**